inc/azure_c_shared_utility/shared_util_options.h

```diff
@@ -25,6 +25,7 @@ extern "C"
     static STATIC_VAR_UNUSED const char* const OPTION_TRUSTED_CERT = "TrustedCerts";
 
     static STATIC_VAR_UNUSED const char* const OPTION_DISABLE_CRL_CHECK = "DisableCrlCheck";
+    static STATIC_VAR_UNUSED const char* const OPTION_CONTINUE_ON_CRL_DOWNLOAD_FAILURE = "AllowCRLFetchFailure";
```

Comment:
Nitpick: The value you've used here value doesn't seem to be consistent with the other values used. Have you considered ```ContinueOnCrlDownloadFailure``` instead?

Reply:
Doh! I renamed it, and my find / replace missed that one.

Thanks.